addons/account/models/account_move.py

```diff
@@ -117,6 +117,21 @@ def _sequence_fixed_regex(self):
         tracking=True,
         default='draft',
     )
+
+    visualization_state = fields.Selection(
+        selection=[
+            ('draft', 'Draft'),
+            ('posted', 'Posted'),
+            ('sent', 'Sent'),
+            ('cancel', 'Cancelled'),
+        ],
```

Comment:
Perhaps we should move the common states in a const ? Like for `PAYMENT_STATE_SELECTION`